test/fixtures/infra/cfn_table.ts

```diff
@@ -0,0 +1,92 @@
+import { CfnTableProps } from "@aws-cdk/aws-dynamodb";
+
+export const CfnTableProperties: CfnTableProps = {
+  keySchema: [
+    {
+      attributeName: "attributeName",
+      keyType: "keyType",
+    },
+  ],
+
+  // the properties below are optional
```

Comment:
this file is a little confusing as you have this comment in a couple of places so I'm not sure what is optional and what isn't.
e.g. from this comment is `attributeDefinitions` optional but `globalSecondaryIndexes` isn't?

Reply:
So this file is generated from the AWS docs of what a Cloudformation DynamoDB Table and just used as a test fixture